packages/devtools_app/lib/src/screens/app_size/app_size_screen.dart

```diff
@@ -590,3 +629,17 @@ extension DiffTreeTypeExtension on DiffTreeType {
     }
   }
 }
+
+extension AppUnitExtension on AppUnit {
+  String get display {
```

Comment:
I think you can do this on the enum directly now with enhanced enums

Reply:
Good point, done!